e2e/parseError.js

```diff
@@ -5,6 +5,50 @@ const ERC7412_ABI = [
   'error OracleDataRequired(address oracleContract, bytes oracleQuery)',
   'error FeeRequired(uint feeAmount)',
 ];
+const PYTH_ERRORS = [
+  // Function arguments are invalid (e.g., the arguments lengths mismatch)
+  // Signature: 0xa9cb9e0d
+  'error InvalidArgument()',
```

Comment:
This is super helpful

Reply:
Awesome yeah! Maybe add erc7412 errors too?